featuretools/entityset/entityset.py

```diff
@@ -577,7 +577,10 @@ def normalize_entity(self, base_entity_id, new_entity_id, index,
         transfer_types = {}
         transfer_types[index] = type(base_entity[index])
         for v in additional_variables + copy_variables:
-            transfer_types[v] = type(base_entity[v])
+            if type(base_entity[v]) == vtypes.DatetimeTimeIndex:
```

Comment:
we should also handle and copying `NumericTimeIndex` variable needing to convert to `Numeric` case